driver-mapping/src/test/java/com/datastax/driver/mapping/MapperAccessorParamsTest.java

```diff
@@ -85,6 +131,43 @@ void should_include_enumtype_in_accessor_string() {
         ResultSet addUserStr(int key, @Enumerated(EnumType.STRING) Enum value);
     }
 
+    /** Tests various ways to match method parameters to query bind markers. */
+    @Accessor
+    public interface UserPhoneAccessor {
+        /** Standard positional markers */
+        @Query("update user set home_phone = ?, work_phone = ? where key = ?")
+        void updatePhones_positional(String homePhone, String workPhone, int key);
+
+        /** Standard named markers */
+        @Query("update user set home_phone = :home, work_phone = :work where key = :key")
```

Comment:
The use of named parameters causes this to fail for 1.2.x, since the Mapper is not meant to support 1.2 anyways should we add @CassandraVersion(major=2.0) for this class?